### ISSUE TITLE
单机单卡用KITTI数据训练IA-SSD，模型收敛后的效果差

### ISSUE THREAD

**Issue by Poet-LiBai** [PaddlePaddle/Paddle3D#246]
### iassd_kitti.yaml配置文件区别
-  由于没有下载kitti的road_plane数据，修改了use_road_plane: False
-  自己用一张3060显卡训练，batch_size为8，而Paddle3D官方是4张V100训练，总的batch_size为4 x 8 = 32，所以我要修改total_step: 37120，Paddle3D官方默认的是 9280，这样我训练的learning_rate曲线趋势才跟官方曲线匹配。

### 训练日志对比
- Paddle3D官方训练日志 第 1 epoch 后loss就由7.5 降到2.32，到第 80 epoch时 loss 降到0.4左右
- 自己训练日志 第 1 epoch 后loss就由7.5 降到6.9，到第 80 epoch时 loss 才降到3.4左右，跟官方差了很远。
- 官方的ipynb示例[【自动驾驶实战】基于Paddle3D&Apollo的点云3D目标物检测](https://aistudio.baidu.com/aistudio/projectdetail/5268894) 也说"模型训练 注意：请确保模型在4卡环境下训练，否则模型效果可能会比较差" 这是什么原因？

### 自己训练评估
2023-02-12 12:58:17,312 -     INFO - [##################################################] 100.00%
2023-02-12 12:58:48,074 -     INFO - Car:
2023-02-12 12:58:48,074 -     INFO - BBOX AP_R40@70%: 55.31 48.19 43.55
2023-02-12 12:58:48,074 -     INFO - BEV  AP_R40@70%: 38.64 35.38 30.26
2023-02-12 12:58:48,074 -     INFO - 3D   AP_R40@70%: 16.92 16.64 14.24
2023-02-12 12:58:48,074 -     INFO - BBOX AP_R40@70%: 55.31 48.19 43.55
2023-02-12 12:58:48,074 -     INFO - BEV  AP_R40@50%: 80.36 71.17 63.90
2023-02-12 12:58:48,074 -     INFO - 3D   AP_R40@50%: 75.66 65.47 58.33
2023-02-12 12:58:48,075 -     INFO - Pedestrian:
2023-02-12 12:58:48,075 -     INFO - BBOX AP_R40@50%: 6.17 5.55 5.61
2023-02-12 12:58:48,075 -     INFO - BEV  AP_R40@50%: 3.99 3.95 3.86
2023-02-12 12:58:48,075 -     INFO - 3D   AP_R40@50%: 2.42 2.51 2.29
2023-02-12 12:58:48,075 -     INFO - BBOX AP_R40@50%: 6.17 5.55 5.61
2023-02-12 12:58:48,075 -     INFO - BEV  AP_R40@25%: 10.36 9.64 9.14
2023-02-12 12:58:48,075 -     INFO - 3D   AP_R40@25%: 10.23 9.50 9.04
2023-02-12 12:58:48,075 -     INFO - Cyclist:
2023-02-12 12:58:48,075 -     INFO - BBOX AP_R40@50%: 14.49 13.20 12.84
2023-02-12 12:58:48,075 -     INFO - BEV  AP_R40@50%: 8.66 6.54 6.67
2023-02-12 12:58:48,075 -     INFO - 3D   AP_R40@50%: 8.12 5.60 5.72
2023-02-12 12:58:48,075 -     INFO - BBOX AP_R40@50%: 14.49 13.20 12.84
2023-02-12 12:58:48,075 -     INFO - BEV  AP_R40@25%: 14.65 13.54 13.31
2023-02-12 12:58:48,076 -     INFO - 3D   AP_R40@25%: 14.60 13.05 12.71
2023-02-12 12:58:48,076 -     INFO - Car:
2023-02-12 12:58:48,076 -     INFO - BBOX AP_R11@70%: 56.82 49.67 46.77
2023-02-12 12:58:48,076 -     INFO - BEV  AP_R11@70%: 39.59 37.44 32.30
2023-02-12 12:58:48,076 -     INFO - 3D   AP_R11@70%: 18.33 19.04 16.90
2023-02-12 12:58:48,076 -     INFO - BBOX AP_R11@70%: 56.82 49.67 46.77
2023-02-12 12:58:48,076 -     INFO - BEV  AP_R11@50%: 79.13 71.42 64.03
2023-02-12 12:58:48,076 -     INFO - 3D   AP_R11@50%: 73.57 65.07 57.31
2023-02-12 12:58:48,076 -     INFO - Pedestrian:
2023-02-12 12:58:48,076 -     INFO - BBOX AP_R11@50%: 12.94 12.48 12.44
2023-02-12 12:58:48,076 -     INFO - BEV  AP_R11@50%: 11.46 10.96 10.86
2023-02-12 12:58:48,076 -     INFO - 3D   AP_R11@50%: 3.70 4.21 4.16
2023-02-12 12:58:48,077 -     INFO - BBOX AP_R11@50%: 12.94 12.48 12.44
2023-02-12 12:58:48,077 -     INFO - BEV  AP_R11@25%: 16.80 16.19 15.05
2023-02-12 12:58:48,077 -     INFO - 3D   AP_R11@25%: 16.66 16.13 14.95
2023-02-12 12:58:48,077 -     INFO - Cyclist:
2023-02-12 12:58:48,077 -     INFO - BBOX AP_R11@50%: 18.99 18.47 17.78
2023-02-12 12:58:48,077 -     INFO - BEV  AP_R11@50%: 14.33 13.43 13.39
2023-02-12 12:58:48,077 -     INFO - 3D   AP_R11@50%: 13.78 12.73 12.43
2023-02-12 12:58:48,077 -     INFO - BBOX AP_R11@50%: 18.99 18.47 17.78
2023-02-12 12:58:48,077 -     INFO - BEV  AP_R11@25%: 19.49 18.72 18.87

**Comment by Birdylx** [comment]
bs会影响模型的收敛精度，你需要根据linear-rule同时调整学习率

**Comment by nepeplwu** [comment]
该issue较长时间无反馈，先关闭，如有问题请重新打开或者建立新的issue
